docs/architecture/architecture.tex

```diff
@@ -100,10 +113,37 @@ \subsection{Programming languages and auxiliary programs}
 
 \subsection{Used libraries}
 \begin{description}
-\item[JavaFX] Creating the graphical user interface.
-\item[Mockito] Mocking objects to test behavior.
-\item[Apache POI] Reading data analysis files stored in the Microsoft Excel format.
-\item[Java SAX] Reading the xml file that specify the datafiles.
+
+\item[JavaFX] Creating the graphical user interface. We chose JavaFX since this
+is Java's latest framework to create these interfaces and provides some
```

Comment:
Java's latest framework. Ik denk dat er latere zijn, maar fx zit standaard in java8 dat was de voornaamste reden en dat sommige minimale ervaring met fx hadden